packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -24,18 +24,17 @@ use error_stack::Report;
 use futures::TryFutureExt;
```

Comment:
(comment location irrelevant)
I assume there were no merge conflicts?
Locally i am able to cherry pick the link commit with no merge conflicts. Maybe be easier to just rebase the commit onto main on the dev/links branch and force push than having this merge commit?

Reply:
No merge conflicts. We can discuss the rebase strategy but considering we're going to merge from now on it might be better to just be consistent from the get go

Reply:
Sure, no worries then. I don't think we should review all changes then